app/controllers/TwoEyeReviewController.scala

```diff
@@ -42,7 +42,7 @@ class TwoEyeReviewController @Inject() (
 
   val logger = Logger(getClass)
```

Comment:
Couple of these where type annotation is requested in IntelliJ - no problem if we do not want to supply it - just an observation

Reply:
added now